packages/blocks/src/api/registration.js

```diff
@@ -126,6 +133,16 @@ import { store as blocksStore } from '../store';
  *                                              then no preview is shown.
  */
 
+/**
+ * A subset of `WPBlock` type. Contains basic block's information for display reasons.
```

Comment:
It feels more like part of `@wordpress/block-editor` package. Are you sure it should be promoted as a subset of `WPBlock` type? Isn't it a type that exists only for UI purposes that is completely different?

Reply:
You're probably right. I'm thinking of naming it differently, move it to `block-editor` and have it return the `attributes` and `blockType` as well. That would simplify the usage of `getBlockLabel` at least in `BlockTitle` and maybe in more places when we'll check this `label` functionality`.

What do you think of having it return these extra props (attributes,blockType)?

Reply:
I don't know much about `getBlockLabel`, can we leave it out for now?

Reply:
Yeah, sure!